cmd/minikube/cmd/start.go

```diff
@@ -141,6 +145,15 @@ func runStart(cmd *cobra.Command, args []string) {
 		glog.Errorln("Error starting the VM: ", err)
 		os.Exit(1)
 	}
+
+	// Set Proxy to as Shell Env
+	if viper.IsSet("http-proxy") || viper.IsSet("https-proxy") {
```

Comment:
> It's required because many time cert creation for ssh need time and that time it do the retry (Which I observed some time back).

And why can we not wait for the cert creation to be finished. Something does not sounds right here. And if that is so, we might have to bite into the sour apple and have the registration in the loop. Correctness wins in this case.